ros_buildfarm/ci_job.py

```diff
@@ -290,6 +290,8 @@ def _get_ci_job_config(
         'underlay_source_paths': underlay_source_paths,
         'trigger_timer': trigger_timer,
         'trigger_jobs': trigger_jobs,
+
+        'show_images': build_file.show_images or {},
```

Comment:
The `or {}` doesn't seem to be necessary if the default value is already an empty dict?